x-pack/legacy/plugins/uptime/public/state/actions/ui.ts

```diff
@@ -6,22 +6,38 @@
 
 export const SET_INTEGRATION_POPOVER_STATE = 'SET_INTEGRATION_POPOVER_STATE';
 export const SET_BASE_PATH = 'SET_BASE_PATH';
+export const REFRESH_APP = 'REFRESH_APP';
 
 export interface PopoverState {
   id: string;
   open: boolean;
 }
 
+interface SetBasePathAction {
+  type: typeof SET_BASE_PATH;
+  payload: string;
+}
+
 interface SetIntegrationPopoverAction {
   type: typeof SET_INTEGRATION_POPOVER_STATE;
   payload: PopoverState;
 }
 
+interface TriggerAppRefreshAction {
+  type: typeof REFRESH_APP;
+  payload: number;
+}
+
 interface SetBasePathAction {
   type: typeof SET_BASE_PATH;
   payload: string;
 }
 
```

Comment:
SetBasePathAction duplicated, i guess side effect of conflicts.

Reply:
Yes, there should only be one

Reply:
Resolved in https://github.com/elastic/kibana/pull/48035/commits/27966f899ca40f1f8191756d893476d12ef59659.